src/Discord.Net.WebSocket/DiscordSocketClient.cs

```diff
@@ -1778,7 +1778,33 @@ private async Task ProcessMessageAsync(GatewayOpCode opCode, int? seq, string ty
                                     }
                                 }
                                 break;
+                            case "INTERACTION_CREATE":
+                                {
+                                    await _gatewayLogger.DebugAsync("Received Dispatch (INVITE_DELETE)").ConfigureAwait(false);
```

Comment:
wrong log

Reply:
Yes I just saw that, I'm going to update it soon